tracer/test/Datadog.Trace.Security.Unit.Tests/WafCompatibilityTests.cs

```diff
@@ -18,21 +17,11 @@ public class WafCompatibilityTests
         [SkippableTheory]
         [InlineData("1.3.0")]
         [InlineData("1.10.0")]
-        public void ShouldNotInitializeWithExportsMissing(string version)
-        {
-            var libraryInitializationResult = WafLibraryInvoker.Initialize(version);
-            libraryInitializationResult.ExportErrorHappened.Should().BeTrue();
-            libraryInitializationResult.Success.Should().BeFalse();
-            libraryInitializationResult.WafLibraryInvoker.Should().BeNull();
-        }
-
-        [SkippableTheory]
-        [InlineData("1.16.0")]
         [InlineData("1.14.0")]
```

Comment:
why remove `ShouldNotInitializeWithExportsMissing`? maybe we could add in the theory the enum member we expect for load fail

Reply:
The problem is all the old versions failed as they were missing the new functions, so separating the test lost meaning.